README.md

```diff
@@ -1,4 +1,6 @@
 # Massa-as-sdk
+![check-code-coverage](https://img.shields.io/badge/coverage-70%25-green)
```

Comment:
looking at the CI file, 70% is suppose to but color orange

Reply:
I made the update about the color after the ```if (oldCoverage != coverage || !oldCoverage) {``` So normally It will update when the coverage will change. I'm working on test in an other branch so it must change soon.